scripts/get-discord-rpc

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
+
+set -e
 
 if [[ $(uname) == "Linux" ]]; then
```

Comment:
I noticed you changed exactly same code in `check-code-formatting`, but not here, any reasoning?

Reply:
This line doesn't need to be changed.